Notification templates (#190)
OHS-Hosting-Infrastructure/infrastructure: docs/how-we-work/notification-templates/initial-message.md

The following should be used as a message template when sending notifications about system outages and this is the first email letting people know.

## When to Use

> If an issue is determined to be P0 or P1 based on docs/how-we-work/support-process.md

Reviewer: ```suggestion
If an issue is determined to be P0 or P1 based on [our Support Process doc definition](support-process.md).
```
